.github/workflows/label-all-new-issues.yml

```diff
@@ -0,0 +1,21 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Label issues
+on:
+  issues:
+    types:
+      - opened
+jobs:
+  label_issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Label issues
+        uses: andymckay/labeler@5c59dabdfd4dd5bd9c6e6d255b01b9d764af4414
```

Comment:
Is there a reason why we want this commit over the latest tag? https://github.com/andymckay/labeler/releases/latest

Reply:
I picked out what was in the recommended doc of how to add labels, though I realize now that we should do reopened as well.

Reply:
It is best practice for security since you know which version of the code the action runs. If using latest and the action repo is  hacked, we could have issue losing our token/access to fleet repo. See https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions